get-started/the-stack.md

```diff
@@ -7,97 +7,147 @@ mapped_pages:
 
-#### APM [stack-components-apm]
+### {{es}} clients [stack-components-elasticsearch-clients]
 
-Elastic APM is an application performance monitoring system built on the {{stack}}. It allows you to monitor software services and applications in real-time, by collecting detailed performance information on response time for incoming requests, database queries, calls to caches, external HTTP requests, and more. This makes it easy to pinpoint and fix performance problems quickly. [Learn more about APM](/solutions/observability/apm/index.md).
+The clients provide a convenient mechanism to manage API requests and responses to and from {{es}} from popular languages such as Java, Ruby, Go, Python, and others.
+Both official and community contributed clients are available.
 
-#### {{beats}} [stack-components-beats]
+[Learn more about the {{es}} clients](/reference/elasticsearch-clients/index.md).
 
-{{beats}} are data shippers that you install as agents on your servers to send operational data to {{es}}. {{beats}} are available for many standard observability data scenarios, including audit data, log files and journals, cloud data, availability, metrics, network traffic, and Windows event logs. [Learn more about {{beats}}](beats://reference/index.md).
+## Explore and visualize [_consume]
 
-#### {{es}} ingest pipelines [stack-components-ingest-pipelines]
+Use {{kib}} to explore and visualize the data that's stored in {{es}} and to manage components of the {{stack}}.
+It is also the home for the {{es}}, Elastic {{observability}} and {{elastic-sec}} [solutions](/get-started/introduction.md).
 
-Ingest pipelines let you perform common transformations on your data before indexing them into {{es}}. You can configure one or more "processor" tasks to run sequentially, making specific changes to your documents before storing them in {{es}}. [Learn more about ingest pipelines](/manage-data/ingest/transform-enrich/ingest-pipelines.md).
+### {{kib}} [stack-components-kibana]
 
-#### {{ls}} [stack-components-logstash]
+With {{kib}}, you can:
 
-{{ls}} is a data collection engine with real-time pipelining capabilities. It can dynamically unify data from disparate sources and normalize the data into destinations of your choice. {{ls}} supports a broad array of input, filter, and output plugins, with many native codecs further simplifying the ingestion process. [Learn more about {{ls}}](logstash://reference/index.md).
+- Use **Discover** to interactively search and filter your raw data.
+- Build custom visualizations like charts, graphs, and metrics with tools like **Lens**, which offers a drag-and-drop experience.  
+- Assemble your visualizations into interactive dashboards to get a comprehensive overview of your information.
+- Analyze geospatial data using the powerful **Maps** application.
+- Manage resources such as processors, pipelines, data streams, trained models, and more.
```

Comment:
Don't alerts fall in this space too?

Reply:
The one thing I think we could improve with this list is you tell me all these cool things I could do and no way to let me go and learn about them if I feel like chasing a squirrel.